paddle/phi/api/yaml/legacy_ops.yaml

```diff
@@ -277,6 +277,15 @@
     data_type : x
   backward : conv2d_transpose_grad
 
+- op : conv2d_transpose_bias
+  args : (Tensor x, Tensor filter, int[] strides={1, 1}, int[] paddings={0, 0}, int[] output_padding={}, IntArray output_size={}, str padding_algorithm="EXPLICIT", int groups=1, int[] dilations={1, 1}, str data_format="NCHW")
```

Comment:
Should it also have Tensor bias as input?

Reply:
I tried. But if adding Bias as input here, the InferMata will report error. Seems the args part should contain necessary vars or optional vars with default value. Since the kernel function still checks inputs, I think it's fine here?

Reply:
这里应该加上Bias，然后在infer_meta配置下添加：param: [x,filter,strides,paddings,output_padding,output_size,padding_algorithm,groups,dilations,data_format]用于表达传给Conv2dTransposeInferMeta的参数都有啥（不包括Bias）

Reply:
好的谢谢~我试一试

Reply:
嗯嗯，加了param这个配置后，args里就可以加上Tensor Bias了~

Reply:
请问除了这里还有什么地方需要改动的吗~ @wanghuancoder

Reply:
没有啦~~